### ISSUE TITLE
 No layout views/post.jade found for collection_posts (WARNING: non-Error used)

### ISSUE THREAD

**Issue by orenmizr** [netlify/roots-collections#2]
1. created a new project with the roots command (3.1.0)
2. only installed the roots collection (npm installed roots-collections --save)
3. updated the app.coffee as required (exactly like in the readme)
4. created the posts folder with the exampled md in the readme
5. ran roots (roots watch)
6.  No layout views/post.jade found for collection_posts -> does not work

unlike the first issue this is a vanilla project. only your code. 

am i doing something wrong or there is an issue with this package?

**Comment by biilmann** [comment]
Hi Oren,

Did you make sure to actually add a layout for the posts?

There should be a file called views/post.jade that has the layout for post pages...

**Comment by orenmizr** [comment]
yes. i have a posts folder with your example markdown post. 

i also have a post.jade with your example jade
and uses the untoched default layout  jade

app.coffee

```
extensions: [
    collections(folder: 'posts', layout: 'post'),
```

posts/post/2015-05-04-this-is-a-blog-post.md

```
---
title: This is the title
desc: A description
----

# I am markdown

Hello
```

views/post.jade (layout jade is default)

```
extends layout

block content
  h3.post-title= entry.title
  .post-body!= entry.body
```

lost a lot on time on this :/

**Comment by biilmann** [comment]
Can you give me access to the repo? Then I'll have a look - this works in the test repos I have..

**Comment by biilmann** [comment]
By the way, you can set the layout when configuring the extension - so if you just want post.jade to extend the default layout, you should also be able to just do:

```
extensions: [
    collections(folder: 'posts', layout: 'layout'),
```

And that would use `views/layout.jade` as the post layout

**Comment by orenmizr** [comment]
off course. it's the basic setup with your plugin installed

https://github.com/orenmizr/billroots :)

**Comment by biilmann** [comment]
Did this:

``` bash
git clone https://github.com/orenmizr/billroots.git
cd billroots/
npm install
roots compile
```

And it worked. I'm on OS X and `roots -v` shows `3.1.0`

**Comment by orenmizr** [comment]
interesting... i am on windows 10 roots --v 3.1.0

is there a readfile command that may not work on windows / folder structure ?

how can i set up my intelliJ debug mode for roots. i could not get my IDE to break on your index.coffee

**Comment by biilmann** [comment]
Ahhh! Now I know what the problem is. Got a meeting now, but will push a fix as soon as I'm done...

**Comment by orenmizr** [comment]
10X!

**Comment by biilmann** [comment]
Published version `0.3.2` now, and this Windows compatibility issue should be fixed now :) Thanks for reporting it!

**Comment by orenmizr** [comment]
it now seem to work! i am still playing around with it. i could probably give better feedback if i could setup my webstorm to debug roots node operation. i can run via the ide but not debug it. if you are familiar with it, i'd be glad to hear of a way.
